.github/workflows/prerelease.yml

```diff
@@ -0,0 +1,41 @@
+name: PreRelease
+
+on: 
+  workflow_dispatch:
+
+jobs:
+  Publish:
+    name: PreRelease
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.ADMIN_TOKEN }}
+          ref: ${{ github.ref }}
+          # pulls all commits (needed for lerna / semantic release to correctly version)
+          fetch-depth: '0'
+
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+          registry-url: https://registry.npmjs.org/
+
+      - name: Install Packages
+        run: yarn install
+
+      - name: Build
+        run: yarn build
+
+      - name: Authenticate with Registry
+        run: |
+          yarn logout
+          echo "registry=http://registry.npmjs.org/" >> .npmrc
+          echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" >> .npmrc
+          npm whoami
+        env:
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+      - name: Publish package
+        run: |
+          yarn publish:ci:prerelease --preid $(echo ${GITHUB_REF##*/}) patch
```

Comment:
should we always use `patch`?
Maybe get it as user intpu?

also - what is the difference between `patch` and `prepatch` etc?

Reply:
"pre" appends dash and a string to the version. it doesn't matter when using 'canary' flag.

Reply:
about semanticVersion input - 
@blumamir do you think we should add an input for that?
@nirsky WDYT?

Reply:
I don't see when we would want to release a canary that isn't "patch", it's always internal so conventions don't really matter.
I think it simplifies the trigger a bit, but I don't mind adding it al